CHANGELOG.md

```diff
@@ -5,13 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# Legend
```

Comment:
The new line after the header was missed, also should be H2 probably (`##` instead of `#`).

Reply:
Updated. Thanks